cg/cli/workflow/balsamic/base.py

```diff
@@ -45,13 +45,13 @@ def balsamic(context, case_id, priority, email, target_bed):
     context.obj["hk_api"] = hk.HousekeeperAPI(context.obj)
     context.obj["fastq_handler"] = FastqHandler
     context.obj["gzipper"] = gzip
+    context.obj["lims_api"] = lims.LimsAPI(context.obj)
     scout_api = scoutapi.ScoutAPI(context.obj)
-    lims_api = lims.LimsAPI(context.obj)
     tb_api = tb.TrailblazerAPI(context.obj)
     deliver = DeliverAPI(
         context.obj,
         hk_api=context.obj["hk_api"],
-        lims_api=lims_api,
+        lims_api=context.obj["lims_api"],
```

Comment:
could the lims-api be created under the `workflow` group command instead if more workflows will/are using it?

Reply:
Let's keep it specific for now to keep it more readable (hopefully)